circle/functions

```diff
@@ -576,7 +579,7 @@ chart_package() {
     install_s3cmd || exit 1
 
     # the download will fail for new repos
-    if s3cmd get s3://$AWS_BUCKET/${subpath}/index.yaml $repo_index; then
+    if s3cmd get s3://$AWS_BUCKET/${CHART_BRANCH}/index.yaml $repo_index; then
```

Comment:
`function` charts will also be published at `${CHART_BRANCH}`? I think yes!

Reply:
yes, they will also be published there